dist/s_clang_format

```diff
@@ -8,24 +8,38 @@ trap 'rm -rf $t' 0 1 2 3 13 15
 download_clang_format() {
     version=$1
     # FIXME-WT-10619
-    arch=$(uname -m)
-    if [ ! "$arch" = "x86_64" ]; then
-        echo "$0: unsupported architecture $arch to run clang_format"
-        return 1
-    fi
-
+    # The URLs for binaries below don't follow a common pattern (see the build/build prefixes, -1.gz suffixes)
+    # We should consider setting up a common URL format when we migrate to 12.0.1 so the script can be generalised.
+    arch_and_os="$(uname -m)-$(uname)"
     archive=dist/clang-format.tar.gz
-    sys_os=$(uname)
-    if [ "$sys_os" = "Linux" ]; then
-        curl https://s3.amazonaws.com/boxes.10gen.com/build/clang-format-llvm-"$version"-x86_64-linux-gnu-ubuntu-20.04.tar-1.gz -o $archive
-        tar --strip=1 -C dist/ -xf $archive clang-format-llvm-"$version"-x86_64-linux-gnu-ubuntu-20.04/clang-format && rm $archive
-    elif [ "$sys_os" = "Darwin" ]; then
+
+    # Adding more clang-format binaries requires uploading them to boxes.10gen
+    # You can either get the clang-format binary from the llvm releases page
+    # (https://github.com/llvm/llvm-project/releases) or compile clang-format yourself.
+    # Place the binary in dist/ and confirm that s_clang_format runs correctly, then
+    # tar a folder containing just the clang-format binary with the format:
+    # clang-format-llvm-10.0.0-aarch64-linux-gnu/
+    #     clang-format
+    # This tarball should extract using the tar command below.
+    # The tarball can then be uploaded via a Jira request. See BUILD-17387 for an example.
+    case $arch_and_os in
```

Comment:
`uname -m` gave me `arm64` on my macbook. Should we cover this case as well?

Reply:
I'd prefer to leave this out of the current PR as this ticket unblocks our migration to the new ARM workstations.
I think it'd be useful to do alongside the WT-10619 upgrade to 12.0.1 though.

Reply:
Fair enough